src/components/playback/playmethodhelper.js

```diff
@@ -2,8 +2,7 @@ export function getDisplayPlayMethod(session) {
     if (!session.NowPlayingItem) {
         return null;
     }
-
-    if (session.TranscodingInfo?.IsVideoDirect && session.TranscodingInfo.IsAudioDirect) {
+    if ((!session.TranscodingInfo?.VideoCodec && session.TranscodingInfo.IsAudioDirect) || (session.TranscodingInfo?.IsVideoDirect && session.TranscodingInfo.IsAudioDirect)) {
```

Comment:
Alternatively (not tested):
```suggestion
    if ((session.TranscodingInfo?.IsVideoDirect || !session.TranscodingInfo?.VideoCodec) && session.TranscodingInfo.IsAudioDirect) {
```

Reply:
When audio remuxing the transcodingInfo will look like this:

<img width="399" alt="image" src="https://github.com/user-attachments/assets/def420e9-ebca-4b39-9768-e9e1c856cba1">

But I do agree that we probably want to use `session.TranscodingInfo?` just in case.